content/en/docs/concepts/scheduling-eviction/node-pressure-eviction.md

```diff
@@ -283,20 +328,26 @@ the Pods' relative priority to determine the eviction order, because inodes and
 requests.
 
 The kubelet sorts pods differently based on whether the node has a dedicated
-`imagefs` filesystem:
+`imagefs` or `containerfs` filesystem:
 
 #### With `imagefs`
 
-If `nodefs` is triggering evictions, the kubelet sorts pods based on `nodefs`
-usage (`local volumes + logs of all containers`).
-
-If `imagefs` is triggering evictions, the kubelet sorts pods based on the
-writable layer usage of all containers.
+- If `nodefs` is triggering evictions, the kubelet sorts pods based on `nodefs`
+  usage (`local volumes + logs of all containers`).
+- If `imagefs` is triggering evictions, the kubelet sorts pods based on the
```

Comment:
maybe add a blurb `(assuming imagefs and containerfs are not split)` or something, as in that case writable layer sorting will happen for containerfs

Reply:
oh I see. maybe the title of this section should be "Split Disk (With `imagefs` and no `containerfs`)" or something to make clearer the case

Reply:
Done.